app/components/threads_button/__snapshots__/threads_button.test.tsx.snap

```diff
@@ -190,7 +190,98 @@ exports[`Thread item in the channel list Threads Component should match snapshot
 </View>
 `;
 
-exports[`Thread item in the channel list Threads Component should match snapshot with only unreads filter 1`] = `null`;
+exports[`Thread item in the channel list Threads Component should match snapshot with only unreads filter 1`] = `
```

Comment:
The test that generates this snapshot probably makes no sense now, because the `onlyUnreads` no longer affects anything.